src/PostProcesses/postProcess.ts

```diff
@@ -477,6 +490,71 @@ export class PostProcess {
         this.width = -1;
     }
 
+    private _createRenderTargetTexture(textureSize: { width: number, height: number }, textureOptions: RenderTargetCreationOptions, channel = 0) {
+        for (let i = 0; i < this._textureCache.length; i++) {
+            if (this._textureCache[i].width === textureSize.width &&
+                this._textureCache[i].height === textureSize.height &&
+                this._textureCache[i]._postProcessChannel === channel) {
+                return this._textureCache[i];
+            }
+        }
+
+        const tex = this._engine.createRenderTargetTexture(textureSize, textureOptions);
+        tex._postProcessChannel = channel;
+        this._textureCache.push(tex);
+
+        return tex;
+    }
+
+    private _flushTextureCache() {
+        const currentRenderId = this._renderId;
+
+        for (let i = this._textureCache.length - 1; i >= 0; i--) {
+            if (currentRenderId - this._textureCache[i]._lastUsedRenderId > 100) {

```

Comment:
Same thing for `_lastUsedRenderId` than for `_postProcessChannel`, I think it's better to have:
```typescript
private _textureCache: Array<{texture: InternalTexture, postProcessChannel: number, _lastUsedRenderId : number}> = [];
```
and not make `InternalTexture` grow only for this post process usage.